src/api/app/controllers/webui/request_controller.rb

```diff
@@ -26,7 +26,11 @@ def show
       action_id = params[:request_action_id] || @bs_request.bs_request_actions.first.id
       @action = @bs_request.webui_actions(filelimit: @diff_limit, tarlimit: @diff_limit, diff_to_superseded: @diff_to_superseded,
                                           diffs: true, action_id: action_id.to_i, cacheonly: 1).first
-      @active_action = @bs_request.bs_request_actions.find(action_id)
+      actions = @bs_request.bs_request_actions
+      @active_action = actions.find(action_id)
+      active_action_index = actions.index(@active_action)
+      @prev_action = actions[active_action_index - 1] unless active_action_index.zero?
+      @next_action = actions[active_action_index + 1] if active_action_index + 2 < actions.length
```

Comment:
When you visit the request page `Next` is disabled even if there are two actions in the list. I think the `Next` button should be active when you are standing on 1st action and there are more actions in the dropdown list

Reply:
It's fixed now